compose.yml

```diff
@@ -27,6 +27,8 @@ services:
     build:
       context: .
       args:
+        ENVIRONMENT: dev
```

Comment:
Why are some environment defaults `dev` and some `prod`? (I imagine this is a software engineering design practice, would love to learn more)

Reply:
The `Dockerfile` will be used in production environments.  In some upcoming work, I have a release process that publishes these containers to our ghcr.io package repo.  From there NBS, and hopefully other future STLTs, will download the image and install it in their environments.  From a customer's perspective, defaulting to "prod" makes a lot of sense.

The compose.yml is just something used by developers for testing and getting things working locally.  It's a convenience layer to get things up and running, but never used in production environments.  Theoretically ops teams could use something like a compose.yml to setup cloud and on-prem environments, but I've never seen that done before, everyone these days leans into [K8s](https://kubernetes.io/) or vendor specific solutions like [ECS](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/Welcome.html).